web/styles/subscriptions.css

```diff
@@ -340,34 +340,48 @@ h4.user_group_setting_subsection_title {
     }
 }
 
-/* Subheader styles. These aren't in a separate subheader div yet,
-   so we add `.overlay-container` for now to apppease the linter in
-   differentiating from the main container styles below. */
-.two-pane-settings-container.overlay-container {
+.two-pane-settings-container .two-pane-settings-subheader {
+    display: flex;
+    align-items: center;
+    border-bottom: 1px solid var(--color-border-modal-bar);
+    grid-area: subheader;
+
+    .left,
+    .right {
+        display: flex;
+        justify-content: center;
+        /* TODO(evy): how to get the right panel subheader to fill its full
```

Comment:
These aren't properly grid items, I don't think.

Reply:
I've tried fiddling a bunch with making various things in its ancestry `display: grid` and changing grid settings, but I haven't yet found a way to get this working. Would you be down to poke at it a bit?

Reply:
Hmmmm. Well, because the outer grid defined on `two-pane-settings-container` is really just defining row heights, I wonder if there should just be a shared grid definition across `.two-pane-settings-header`, `.two-pane-settings-subheader`, and `.two-pane-settings-body`? Though at this point, I think I've lost the thread of what you're trying to accomplish here--I'm assuming it's columnar layout/alignment, but perhaps I'm wrong about that?